docs/content/docs/testing/telemetry.md

```diff
@@ -240,24 +242,54 @@ export class ConsoleLogger implements ITelemetryBaseLogger {
-
-    // Get or create the document depending if we are running through the create new flow
-    const client = useAzure ? AzureClient :  new TinyliciousClient();
-    const [fluidContainer, containerServices] = createNew
-        ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
-        : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
+  // Create a custom ITelemetryBaseLogger object to pass into the Tinylicious container
+  // and hook to the Telemetry system
+  const azureConfig = {
+      connection: connectionProps,
+      logger: new ConsoleLogger(),
+  };
+  const client = new AzureClient(azureConfig);
+  let container: FluidContainer;
+  let services: AzureContainerServices;
+  let id: string;
+
+  // Get or create the document depending if we are running through the create new flow
+  const createNew = !location.hash;
+  if (createNew) {
+      // The client will create a new detached container using the schema
+      // A detached container will enable the app to modify the container before attaching it to the client
+      ({container, services} = await client.createContainer(containerSchema));
 ```
 
 Now, whenever a telemetry event is encountered, the custom `send()` method gets called and will print out the entire
 event object.
 
 <img src="https://fluidframework.blob.core.windows.net/static/images/consoleLogger_telemetry_in_action.png" alt="The
   ConsoleLogger sends telemetry events to the browser console for display.">
+
+{{% callout warning %}}
+
+The purpose of `ConsoleLogger` is purely to demonstrate how the `logger` interface should be implemented. The Fluid Framework provides `DebugLogger` by default if custom logger is not provided. For more information, please refer to [Using DebugLogger](#using-debuglogger) below instead of implementing something similar to `ConsoleLogger`.
+
+{{% /callout %}}
+
+### Using `DebugLogger`
+
+The `DebugLogger` offers a conveneient way to output all telemetry events to the console. Like mentioned previously, creating or getting a container creates a `Loader` object, which will then create a `DebugLogger` in its constructor. Hence, `DebugLogger` is already provided by default when creating/getting a container, and no extra steps is required to use the logger.
+
+Under the hood, `DebugLogger` make use of the [debug](https://github.com/visionmedia/debug) library. `Debug` allows a library to log messages to a namespace. By default these messages aren't displayed but can be enabled by the app that is making use of the library. Debug is a popular package used by most major node modules (express, socket.io, etc...). For our node apps enabling library logging is as simple as setting the DEBUG environment variable - i.e.
+
+`DEBUG=fluid:*,connect:compress,connect:session`
+
+This is already done in our docker compose files for our own internal libraries which are all under the routerlicous namespace.
```

Comment:
Not sure if this line is necessary and may just confuse people since we don't discuss routerlicious elsewhere here

Reply:
yup! I took it out from @heliocliu previous suggestion 😊